openml-h2o/src/test/java/com/feedzai/openml/h2o/H2OModelProviderTrainTest.java

```diff
@@ -75,13 +76,12 @@ public void createDataset() {
      * Creates a dataset to be used on the tests, based on the provided schema.
      *
      * @param schema The schema that the dataset will comply to.
+     *
      * @return A new dataset.
      */
     private Dataset createDataset(final DatasetSchema schema) {
         final Map<String, String> params = H2OAlgorithmTestParams.getIsolationForest();
-        final int sampleSize = Optional.ofNullable(params.get("sample_size"))
-                .map(Integer::parseInt)
-                .orElse(256);
+        final int sampleSize = Optional.ofNullable(params.get("sample_size")).map(Integer::parseInt).orElse(256);
```

Comment:
tbh, I find more readable to have each chained operation in a single line, but feel free to keep as it is now